releasenotes/notes/add-num_qubits-qasm3-loads-b7a6e30147e18f52.yaml

```
@@ -0,0 +1,8 @@
+---
+features_qasm:
+  - |
+    The functions :func:`qasm3.loads` and :func:`qasm3.load` now has an extra argument
```

```suggestion
    The functions :func:`.qasm3.loads` and :func:`.qasm3.load` now have an extra argument
```

The leading `.` causes a suffix lookup (the full names of those functions are `qiskit.qasm3.load` etc).